### PR TITLE
Fixes myohClientVersion & config-development.json

### PR DESCRIPTION
**Pull request by marziman**
- Correction of the myohClientVersion to ClientVersion
- Fixes in config-development.json to enable local workspace development

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -901,7 +901,7 @@ io.sockets.on('connection',function(socket){
                 openhab.status = 'online';
                 openhab.last_online = new Date;
                 openhab.openhabVersion = socket.handshake.openhabVersion;
-                openhab.myohVersion = socket.handshake.myohVersion;
+                openhab.clientVersion = socket.handshake.clientVersion;
```
Comment (digitaldan) [inline app.js]: If you change this here, you also need to update the openhab model so it's persisted to the mongo database, also this is a schema change, so we will need to clean out the old entry from records.

File: app.js [hunk 2]
```diff
@@ -916,7 +916,7 @@ io.sockets.on('connection',function(socket){
                 notifyOpenHABStatusChange(openhab, "online");
             } else {
                 openhab.openhabVersion = socket.handshake.openhabVersion;
-                openhab.myohVersion = socket.handshake.myohVersion;
+                openhab.clientVersion = socket.handshake.clientVersion;
```
Comment (digitaldan) [inline app.js]: Same here

File: app.js (hunk 1 above)
Comment (marziman) [inline app.js]: @digitaldan should we "add" it to the mongodb schema or remove the old entry in schema.
I think we can add it or what strategy do we wanna go with this?

Comment (marziman) [inline app.js]: we could also let the myohVersion remain and add the clientVersion on top? Another way is to clean it out.

Comment (digitaldan) [inline app.js]: @marziman sorry for late late response, lets go ahead and update the model and store the correct field name.  Once this is changed we can go back and remove the field from existing documents.  Also did you see openhabaccesslog.js?  is also has a myopenhab reference you will want to change.

File: models/openhab.js [hunk 3]
```diff
@@ -12,6 +12,7 @@ var OpenhabSchema = new Schema({
     account: {type: ObjectId},                          // An account openHAB belongs to
     openhabVersion: String,                             // openHAB version
     myohVersion: String,                                // myopenhab bundle version
```
Comment (digitaldan) [inline models/openhab.js]: @marziman we should remove the other field

File: models/openhabaccesslog.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ var OpenhabAccessLogSchema = new Schema({
     remoteHost: String,
     remoteVersion: String,
     remoteMyohVersion: String,
```
Comment (digitaldan) [inline models/openhabaccesslog.js]: @marziman we should remove the other field

**Comment by digitaldan** [comment]
@marziman have you tested this on your instance and checked that everything looks ok in the mongo db?

**Comment by marziman** [comment]
@digitaldan i will remove the old variable and retest it. Please wait, I will ping you. Thanks.

**Comment by marziman** [comment]
@digitaldan I ve tested it locally and mongo looks fine.
I took out the old fields from schema. From my point it looks ok.

**Pull request merged**